requirements/requirements-all.txt

```diff
@@ -1,4 +1,4 @@
-psycopg2
+psycopg2-binary
```

Comment:
Why the binary version?

Reply:
*Discussed in meeting*: Leave as binary for now. In ~6 months lets revisit and change. See if `psycopg` has any updates